contracts/test-utils/CHANGELOG.json

```diff
@@ -1,4 +1,18 @@
 [
+    {
+        "version": "3.1.0",
+        "changes": [
+            {
+                "note": "Create RichRevertReason classes"
```

Comment:
We sometimes use markdown backticks on identifiers here so that it looks nice on github and discord later.